sidekick/utils.py

```diff
@@ -38,9 +41,19 @@
     "h2ogpt-sql-sqlcoder-34b-alpha": "defog/sqlcoder-34b-alpha",
     "h2ogpt-sql-nsql-llama-2-7B": "NumbersStation/nsql-llama-2-7B",
     "gpt-3.5-turbo": "gpt-3.5-turbo-1106",
-    "gpt-4-8k": "gpt-4",
-    "gpt-4-1106-preview-128k": "gpt-4-1106-preview"
-
+    "gpt-4-8k": "gpt-4", # leaving this for backward compatibility
+    "gpt-4-1106-preview-128k": "gpt-4-1106-preview", # leaving this for backward compatibility
+    "gpt-4o": "gpt-4o",
+    "gpt-4-turbo": "gpt-4-turbo",
+    "gpt-4-turbo-2024-04-09": "gpt-4-turbo-2024-04-09",
+    "gpt-4-turbo-preview": "gpt-4-turbo-preview",
+    "gpt-4-0125-preview": "gpt-4-0125-preview",
+    "gpt-4-vision-preview": "gpt-4o", # legacy to be deprecated
+    "gpt-4-1106-vision-preview": "gpt-4o", # legacy to be deprecated
+    "gpt-4": "gpt-4",
+    "gpt-4-0613": "gpt-4-0613",
+    "gpt-4-32k": "gpt-4-turbo", # legacy to be deprecated
```

Comment:
Questions:
Is there any reason to add vision-related ones? Should we remove those?